ogm/contribute.adoc

```diff
@@ -95,12 +100,9 @@ If you just want to build the documentation only,
 run it from the [filename]+hibernate-ogm-documentation/manual+ subdirectory.
 ====
 
-=== How to contribute code effectively
+=== How to contribute the code effectively
```

Comment:
Is there not a 'to' missing "How to contribute to the code effectively". That said, why not just "How to contribute"